scripts/install/opt/seagate/cortx/motr/libexec/motr_cfg.sh

```diff
@@ -163,19 +163,26 @@ do_m0provision_action()
     local LVM_SIZE=0
 
     msg "Configuring host [`hostname -f`]"
+    MD_DEVICES=($(lvs -o lv_path | grep "lv_raw_metadata" | grep srvnode  | sort -u))
 
-    MD_DEVICE=$(lvs -o lv_path | grep "lv_raw_metadata" | head -1)
-    if [[ $? -eq 0 && $MD_DEVICE != "" ]];then
-        LVM_SIZE=$(lvs $MD_DEVICE  -o LV_SIZE \
-              --noheadings --units b --nosuffix | xargs)
-
-        ANY_ERR=$(echo $LVM_SIZE | grep -i ERROR | wc -l)
-        if [[ ( "$ANY_ERR" != "0" ) || ( -z $LVM_SIZE ) ]]; then
-            err "lvs $MD_DEVICE command failed."
+    for i in ${MD_DEVICES[@]};
+    do
+        LVM_SIZE_ITR=$(lvs $i -o LV_SIZE \
+                       --noheadings --units b --nosuffix | xargs)
+        ANY_ERR=$(echo $LVM_SIZE_ITR | grep -i ERROR | wc -l)
+        if [[ ( "$ANY_ERR" != "0" ) || ( -z "$LVM_SIZE" ) ]]; then
```

Comment:
If there is any error or LVM_SIZE is empty then it will print error and will not go to update LVM_SIZE_MIN (I replaced double if .. with if()  .. elseif ()).  and at last unless LVM_SIZE_MIN has some non zero value we will not update BESEG size.
so I think we are safe here.